src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -402,14 +410,17 @@ static bool checkAndUpdate (BSONObjBuilder& newAttr, BSONObj attr, ContextAttrib
         }
         else
         {
+            //
             // FIXME P6: in the case of compound value, it's more difficult to know if an attribute
             // has really changed its value (many levels have to be traversed). Until we can develop the
             // matching logic, we consider actualUpdate always true.
+            //
             *actualUpdate = true;
+            updated       = true;  // FIXME: is it OK to do this?
```

Comment:
I have changuing mongoBackend in another branch style changes may involve a conflict difficult to solve in the future...

Could you rollback the style changes and re-do the actual change, please?